examples/backendclient/profiles/testprofile.json

```diff
@@ -1,14 +1,51 @@
 {
-   "name":"testprofile",
-   "properties":{
-      "playerPool":{
-         "region.europe-west1":"0-150",
-         "mmr.rating":"950-1650",
-         "mode.ctf":"0-9999999999"
-      },
-      "roster":{
-         "blue":4,
-         "red":4
-      }
+  "name":"testprofilev1",
+  "id":"testprofile",
+  "properties":{
+     "pools": [
+        {
+          "name": "defaultPool",
+          "filters": [
+            { "name": "europeWest1ElapsedUnder150", "attribute": "region.europe-west1", "maxv": "150" },
+            { "name": "silverRanking", "attribute": "mmr.rating", "maxv": "1250", "minv": "950" }
+          ]
+        },
+        {
+          "name": "supportPool",
+          "filters": [
+            { "name": "europeWest1ElapsedUnder150", "attribute": "region.europe-west1", "maxv": "150" },
+            { "name": "silverRanking", "attribute": "mmr.rating", "maxv": "1250", "minv": "950" },
+            { "name": "supportRole", "attribute": "role.support", "maxv": "2147483647" }
+          ]
+        }
+     ],
+     "rosters": [
+        {
+          "name": "red",
+          "players": [
+            { "pool": "defaultPool" },
```

Comment:
After a few reads, this still confuses me a bit, but maybe it's just the example. I think there are three different concepts at play here:
1. A team config - a way to tell my custom match function how to organize and draft my teams
2. A roster - the players that end up in the team-shapes with any metadata they came with... maybe some team data as well
3. The players in the match - so that open match can put them on ignore lists automatically

I get the backfill perspective for something this prescriptive... like: "I already have some players, tell my match function to give a few more"... maybe there's an optimization for open match here, but could still just be custom function though?
**#3** seems like the important one for open match. I get that a developer may want to re-use what #3 is providing... which looks like a landing zone for sticking in teams, but then we have to teach open match what common ways developers may want to construct those teams... e.g. subteams (squads), conditional rule-teams.

Suggestions welcome!